src/optimize.jl

```diff
@@ -236,6 +236,10 @@ function MOI.get(model::Optimizer, ::MOI.TerminationStatus)
         else
             return MOI.OPTIMAL
         end
+    elseif model.solver_status == "UNKNOWN" &&
+           model.time_limit_sec !== nothing &&
+           abs(model.solve_time_sec - model.time_limit_sec) < 1
```

Comment:
```suggestion
           model.solve_time_sec > model.time_limit_sec + 0.1
```

Reply:
Now I take another look, why the `abs`?

Reply:
we can probably get rid of the abs, but not sure if 0.1s is enough. I 've seen solve_time_sec = 1.12s for time_limit_sec = 1s.

Reply:
yes 1s seems too much - agree this is hacky. I 'm open to other ideas

Reply:
I changed to just if the solve time exceeds the limit.